### ISSUE TITLE
老师如何进一步提高验证集准确率

### ISSUE THREAD

**Issue by Ter-hash** [mousecpn/MachineLearning_HW_CQUT#8]

**Comment by mousecpn** [comment]
调参。
有几个方向可以参考。
增大模型表达能力：用pre-trained model、加卷积层等
防止过拟合：dropout、weight decay等
调优化器：lr decay、adam
调loss：focal loss、amsoftmax等
反正调参大法好就是了，19年cvpr有一篇叫调参的叫什么freebies来着，你可以查一下

**Comment by mousecpn** [comment]
> 调参。 有几个方向可以参考。 增大模型表达能力：用pre-trained model、加卷积层等 防止过拟合：dropout、weight decay等 调优化器：lr decay、adam 调loss：focal loss、amsoftmax等 反正调参大法好就是了，19年cvpr有一篇叫调参的叫什么freebies来着，你可以查一下

我讲的是神经网络啊，就算不是神经网络也是调参就好了，炼丹嘛，太上老君都调了几千年了
